packages/browsers/test/src/chrome/install.spec.ts

```diff
@@ -106,6 +107,34 @@ describe('Chrome install', () => {
     assert.deepStrictEqual(browser, installed[0]);
   });
 
+  it('should uninstall a browser', async function () {
```

Comment:
Nit: it feels like this should be in a separate file as we have one for each command so to say.